bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 5.1.6
+version: 6.0.0
```

Comment:
The major version bump isn't necessary now that you are using a backwards compatible approach.